solana/programs/swap-layer/src/utils/relayer_fees.rs

```diff
@@ -118,45 +117,53 @@ pub fn calculate_relayer_fee(
     let mut relayer_fee = u64::from(relay_params.base_fee);
 
     // Calculate the gas dropoff cost in USDC terms.
-    if denorm_gas_dropoff > 0 {
+    if specified_gas_dropoff > 0 {
         require!(
-            denorm_gas_dropoff <= denormalize_gas_dropoff(relay_params.max_gas_dropoff),
+            specified_gas_dropoff <= relay_params.max_gas_dropoff,
             SwapLayerError::InvalidGasDropoff
         );
 
         let gas_dropoff_cost = calculate_gas_dropoff_cost(
-            denorm_gas_dropoff,
+            specified_gas_dropoff,
             relay_params.gas_dropoff_margin,
             relay_params.native_token_price,
         )
         .ok_or(SwapLayerError::GasDropoffCalculationFailed)?;
+
         relayer_fee = relayer_fee.saturating_add(gas_dropoff_cost);
     }
 
     // Compute the relayer fee based on the cost of the relay in the
     // target execution environment's gas units (converted to USDC).
-    let _ = match relay_params.execution_params {
-        ExecutionParams::None => {
-            err!(SwapLayerError::InvalidExecutionParams)
-        }
+    match relay_params.execution_params {
         ExecutionParams::Evm {
             gas_price,
             gas_price_margin,
         } => {
-            let mut total_gas = EVM_GAS_OVERHEAD;
-
-            if denorm_gas_dropoff > 0 {
-                total_gas = total_gas.saturating_add(DROPOFF_GAS_OVERHEAD);
-            }
-
-            let swap_overhead = match output_token {
-                OutputToken::Gas(swap) | OutputToken::Other { address: _, swap } => {
-                    calculate_evm_swap_overhead(&swap.swap_type)
-                        .ok_or(SwapLayerError::EvmGasCalculationFailed)?
-                }
-                _ => 0,
-            };
-            total_gas = total_gas.saturating_add(swap_overhead);
+            // let mut total_gas = EVM_GAS_OVERHEAD;
+
+            let total_gas = EVM_GAS_OVERHEAD
+                .saturating_add(if specified_gas_dropoff > 0 {
+                    DROPOFF_GAS_OVERHEAD
+                } else {
+                    0
+                })
+                .saturating_add(match output_token {
+                    OutputToken::Gas(swap) | OutputToken::Other { address: _, swap } => {
+                        calculate_evm_swap_overhead(&swap.swap_type)
+                            .ok_or(SwapLayerError::EvmGasCalculationFailed)?
+                    }
+                    _ => 0,
+                });
+
+            // let swap_overhead = match output_token {
```

Comment:
Remove this too.

Reply:
Fixed here: https://github.com/wormholelabs-xyz/swap-layer/pull/30/commits/264000226f6942e5e698d1dcffc5cb1dc2bc033e